app/pages/lab-fem/workflow.jsx

```diff
@@ -506,6 +514,29 @@ class EditWorkflowPage extends Component {
 
             <hr />
 
+            {Array.from(this.props.project.experimental_tools).includes('caesarEnabled') ?
+              <div>
+                <p>
+                  <label>
+                    <input
+                      type='checkbox'
+                      checked={isCaesarEnabled}
+                      onChange={this.toggleCaesarEnabled}
+                    />
+                    Caesar Enabled
+                  </label>
+                  <br />
+                  <small className="form-help">A caesar enabled workflow allows the pulling of subject reductions when the subject loads in the classifier.</small>
+                  <br />
+                  <br />
+                  <small className="form-help">If you&apos;d like more help with configuring Caesar, please get in touch via the <a href='/about/contact'>Contact Us</a> page.</small>
```

Comment:
```suggestion
                  <small className="form-help">Enabling Caesar data fetching allows the website to pull in subject reductions when the subject loads in the classifier.</small>
```

Reply:
I'm suggesting that we keep the description short-and-sweet under the assumption the project teams will already have to be in touch with Zoo team in order to get the experimental feature activated via the admin panel.